### PR TITLE
[CELEBORN-999] MR deps check

### PR DESCRIPTION
**Pull request by cxzl25**
### How was this patch tested?
```
./dev/dependencies.sh  --module mr --check
./dev/dependencies.sh  --module mr --check --sbt
```

### CODE REVIEW

File: dev/dependencies.sh [hunk 1]
```diff
@@ -186,6 +186,10 @@ case "$MODULE" in
     MVN_MODULES="client-flink/flink-1.17"
     SBT_PROJECT="celeborn-client-flink-1_17"
     ;;
+  "mr")
+    MVN_MODULES="client-mr/mr"
+    SBT_PROJECT="celeborn-client-mr"
```
Comment (cxzl25) [inline dev/dependencies.sh:191]: Wait https://github.com/apache/incubator-celeborn/pull/1930

File: project/CelebornBuild.scala [hunk 2]
```diff
@@ -895,6 +906,8 @@ object MRClientProjects {
       .settings(
         commonSettings,
         libraryDependencies ++= Seq(
+          Dependencies.commonsCompress,
+          Dependencies.commonsLogging,
```
Comment (cxzl25) [inline project/CelebornBuild.scala:910]: https://www.scala-sbt.org/release/docs/Library-Management.html#Conflict+Management

> By default, the latest revision is selected

**Comment by waitinfuture** [comment]
Hi @FMX , could you take a look at this?

File: project/CelebornBuild.scala (hunk 2 above)
Comment (cfmcgrady) [reply project/CelebornBuild.scala:910]: I had a discussion with @pan3793 about how to pin dependency versions, and we ultimately chose the `dependencyOverrides` approach. see https://github.com/apache/incubator-celeborn/pull/1831#discussion_r1303760048. cc @pan3793

**Review by cfmcgrady — APPROVED** [review]

**Comment by cfmcgrady** [comment]
thanks, merging to main(v0.4.0)

**Pull request closed without merging**